listenbrainz/tests/integration/test_feedback_api.py

```diff
@@ -666,12 +666,17 @@ def test_get_feedback_for_recordings_for_user_invalid_user(self):
         self.assertEqual(response.json["error"], "Cannot find user: nouser")
 
     def test_get_feedback_for_recordings_for_user_no_recordings(self):
-        """ Test to make sure that the API sends 400 if params recordings has nothing. """
+        """ Test to make sure that the API sends 400 if params recordings is not paased or has nothing. """
```

Comment:
Passed?

Reply:
".. If param `recordings` is not passed or is empty"

Reply:
Makes sense